authorreview.Rmd

```diff
@@ -1,8 +1,18 @@
-# Responding to Editor's review {#respond}
+# Responding to Editor's Review {#respond}
 
-these are placeholder headings
+```{block, type='summaryblock', echo=TRUE}
+This chapter outlines how an author of a blog post or tech note can find and respond to comments from an editor in the GitHub pull request review interface.
+```
 
-## Where to look for comments and suggested commits
+Once you have [submitted](#submitpost) your blog post or tech note, an editor will review your post providing comments on both [content](#content) (does it make sense?) and [technical](#technical) (does it follow our Style Guide?) aspects.
 
+Browse [Chapter 5.1 Review a post](#review) to see how editors use the well documented [GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) to suggest edits, make comments, and approve or request changes to your pull request.
 
-## How to edit and signal to editor you have addressed all comments
+To respond to an editor's review:
+
+- Respond to every inline comment with a thumbs up or thumbs down emoji so the editor knows you have seen it.
+- It's ok to explain why you disagree with some comments.
+- [_Resolve conversation_](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations) for items you have addressed.
```

Comment:
Question: why would I both resolve and add a thumbs up/down?

Reply:
believe it or not, people resolve conversations without making suggested changes and then indicate they did not see them

Reply:
Ah ok!